stag-library-compiler/src/main/java/com/vimeo/stag/processor/generators/AdapterGenerator.java

```diff
@@ -77,4 +84,12 @@ static String getJsonName(@NotNull Element element) {
      */
     @NotNull
     public abstract TypeSpec createTypeAdapterSpec(@NotNull StagGenerator stagGenerator);
+
+
+    FieldSpec createTypeTokenSpec(@NotNull TypeMirror typeMirror){
```

Comment:
Since this is part of the public (well package local) API of this class, it would be good to have javadocs on it.

Reply:
Also could be annotated NotNull

Reply:
Could you run the formatter on this file?